raiden_contracts/constants.py

```diff
@@ -98,7 +98,7 @@ class MessageTypeId(IntEnum):
 START_QUERY_BLOCK_KEY = 'DefaultStartBlock'
 
 
-class ChainId(Enum):
+class ChainId(IntEnum):
```

Comment:
As discussed in rocketchat perhaps let's remove this `IntEnum` and replace it with constants. Semantically this is only the known chain IDs. The user may provide a custom chain ID and then an enum no longer works. This already has bitten us in the client and had to go around it.

Reply:
I left only `ID_TO_NETWORKNAME` and `NETWORKNAME_TO_ID`. As far as I see, these should be enough.